integration-tests/emulated/tests/assets/asset-hub-kusama/src/tests/swap.rs

```diff
@@ -150,8 +150,8 @@ fn swap_locally_on_chain_using_foreign_assets() {
 	let penpal_as_seen_by_ah = AssetHubKusama::sibling_location_of(PenpalA::para_id());
 	let sov_penpal_on_ahk = AssetHubKusama::sovereign_account_id_of(penpal_as_seen_by_ah);
 	AssetHubKusama::fund_accounts(vec![
-		(AssetHubKusamaSender::get().into(), 5_000_000 * KUSAMA_ED), /* An account to swap dot
-		                                                              * for something else. */
+		(AssetHubKusamaSender::get(), 5_000_000 * KUSAMA_ED), /* An account to swap dot
```

Comment:
```suggestion
		(AssetHubKusamaSender::get(), 5_000_000 * ASSET_HUB_KUSAMA_ED), /* An account to swap dot
```

I know that this is kind of random value, but maybe better to use `ASSET_HUB_KUSAMA_ED` for AssetHubKusama instead KUSAMA_ED to avoid confusion why we use KUSAMA_ED for AssetHub? I would fix this also other places, if any.